vos/vos/tests/test_vos.py

```diff
@@ -78,6 +79,38 @@ def test_get_node_url():
     assert kwargs['params']['view'] == 'header'
 
 
+@patch('vos.vos.os.path.exists', Mock())
+def test_rename_vospace_resource():
+    with warnings.catch_warnings(record=True) as w:
+        # Cause all warnings to always be triggered.
+        warnings.simplefilter("always")
+        with patch('vos.vos.open') as open_mock:
+            old_content = 'blah'
+            new_config_mock = Mock()
+            open_mock.return_value.read.return_value = old_content
+            open_mock.return_value.write = new_config_mock
+            vos._rename_vospace_resource()
+        assert new_config_mock.called_once_with(old_content)
+        # nothing changed so no warnings
+        assert len(w) == 0
+
+    new_config_mock.reset_mock()
+    with warnings.catch_warnings(record=True) as w:
+        # Cause all warnings to always be triggered.
```

Comment:
This tests makes sure the warning triggers when we re-write the resource_id but that warning is a bit 'not needed' I think.   But the warning on Connection going to the old resources_id is needed and I don't think this test checks to make sure that warning is being sent?